taperole/hosts

```diff
@@ -1,6 +1,6 @@
 # Base Example
 [production]
-162.243.113.173 be_app_env=production be_app_branch=deployment
+162.243.113.173 be_app_env=production be_app_branch=bugfix/api-keys
```

Comment:
I was working on a different branch and needed to make sure the things I changed were working. I'll change it to master when everything's merged.